src/UserManagerSettings.ts

```diff
@@ -18,7 +18,6 @@ export interface UserManagerSettings extends OidcClientSettings {
     /**
      * The features parameter to window.open for the popup signin window. By default, the popup is
      * placed centered in front of the window opener.
-     * (default: "location=no,toolbar=no,width=576,height=640")
```

Comment:
why is the default removed, in the code we still use a default...
+
comment needs to be adapted as it is an object now:
(default: { location: false, .... })